tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/ui/RemoveOldWatchConsoleListener.java

```diff
@@ -50,7 +52,7 @@ public void consolesAdded(IConsole[] consoles) {
 				if ("GradleConsole".equals(consoleClass.getSimpleName())) {
 					String consoleName = console.getName();
 
-					if (consoleName.contains("watch")) {
+					if (consoleName.startsWith(LiferayGradleWorkspaceProject.WATCH_LAUNCH_CONFIGURATION_NAME)) {
```

Comment:
This constant shouldn't  be in this class.  The "RemoveOldWatchConsoleListener" class shouldn't know anything about "LiferayGradleWorkspaceProject".  So lets refactor this to a constants class.